seth/util.go

```diff
@@ -457,3 +458,24 @@ This error could be caused by several issues. Please try these steps to resolve
 Original error:`
 	return fmt.Errorf("%s\n%s", message, err.Error())
 }
+
+func mustSafeInt64(input uint64) int64 {
+	if input > math.MaxInt64 {
+		panic(fmt.Errorf("uint64 %d exceeds int64 max value", input))
+	}
+	return int64(input)
+}
+
+func mustSafeInt(input uint64) int {
+	if input > math.MaxInt {
+		panic(fmt.Errorf("uint64 %d exceeds int max value", input))
+	}
+	return int(input)
+}
+
+func mustSafeUint64(input int64) uint64 {
+	if input < 0 {
+		panic(fmt.Errorf("int64 %d exceeds uint64 max value", input))
```

Comment:
Confusing error: should tell me it's an underflow, not overflow

Reply:
true!